modules/install/Classes/Install.php

```diff
@@ -657,7 +657,10 @@ public function envcheck(array $configuration)
         if (!$currentMysqlVersion) {
             $mysqlVersionCheck = $not_possible . t('Konnte MySQL-Version nicht überprüfen, da keine Verbindung mit den Standarddaten (%1@%2) möglich war. <br/>Dies ist kein direkter Fehler, bedeutetet aber, dass einige Setup-Schritte per Hand durchgeführt werden müssen. <br/>Bitte Stelle sicher, dass du MySQL mindestens in Version %3 benutzt.', $configuration['database']['user'], $configuration['database']['server'], \LANSUITE_MINIMUM_MYSQL_VERSION);
         } elseif (str_contains($currentMysqlVersion, 'MariaDB')) {
-            $currentMariaDBVersion = substr($currentMysqlVersion, strpos($currentMysqlVersion, '-')+1);
+            $pos = strpos($currentMysqlVersion, '-');
```

Comment:
```suggestion
            $currentMariaDBVersion = $currentMysqlVersion;
            $pos = strpos($currentMysqlVersion, '-');
```

I am setting the `$currentMysqlVersion` as the default maria db version to ensure this works, even if the maria db version has no `-` in it.